src/plugin.py

```diff
@@ -30,6 +30,28 @@
 
 
 class BNetPlugin(Plugin):
+    async def get_game_time(self, game_id, context):
+        gameTimeMinutes = 0
+        log.info('game_id: ' + game_id)
+        if game_id == "5272175": # Overwatch
+            log.debug("Fetching playtime for Overwatch...")
+            player_data = await self.backend_client.get_ow_player_data()
+            if player_data == None:
+                log.error('No Overwatch profile found.')
+                return GameTime(game_id, gameTimeMinutes, None)
+            if player_data['private'] == True:
+                log.info('Unable to get data as Overwatch profile is private.')
+                return GameTime(game_id, gameTimeMinutes, None)
+            qpTime = player_data['playtime']['quickplay']
+            if qpTime.count(':') == 1: # minutes and seconds
+                match = re.search('(?:(?P<m>\\d+):)(?P<s>\\d+)', qpTime)
+                gameTimeMinutes = int(qpTime.group('m'))
+            elif qpTime.count(':') == 2: # hours, minutes and seconds
+                match = re.search('(?:(?P<h>\\d+):)(?P<m>\\d+)', qpTime)
+                gameTimeMinutes = int(match.group('h')) * 60 + int(match.group('m'))
+            log.info('Overwatch quickplay playtime [minutes]: ' + str(gameTimeMinutes) + " (" + qpTime + ")")
+        return GameTime(game_id, gameTimeMinutes, None)
```

Comment:
Please do not return 0 game time for other games. Do like in @elkangaroo proposal https://github.com/bartok765/galaxy_blizzard_plugin/pull/7/files/0f1c35f92e388dceafab74bb93a6335927a8d0fb#r386485266

Reply:
And a minor style thing: don't use `camelCase`, but `snake_case` in variables `gameTimeMinutes`, `qpTtime`...